src/components/FullNameAutoCompleteInput.tsx

```diff
@@ -71,18 +73,38 @@ export const FullNameAutoCompleteInput = ({
       : selectedPersonIndex - 1;
   };
 
-  const handleArrowNavigation = event => {
+  const handleScroll = () => {
```

Comment:
Zda sa ze to funguje fajn, ale nemalo by toto byt po spravnosti v `useEffect` kedze je to side effect componentu?

Reply:
Neviem. Tato metoda sa vola pri stlaceni sipky hore alebo dole (v `onKeyDown`). Asi nechapem, ale ako by sa to dalo napisat s `useEffect`?